src/reporter/health.py

```diff
@@ -1,20 +1,25 @@
-import os
-
 from geocoding import geocoding
 from geocoding.factory import get_geo_cache, is_geo_coding_available
 from cache.factory import get_cache, is_cache_available
+from translators.factory import CRATE_BACKEND, TIMESCALE_BACKEND, \
+    default_backend
 
 
-# TODO having now multiple backends, health check needs update
-def check_crate():
+def check_db(db=CRATE_BACKEND):
```

Comment:
this change is already in #449, so perhaps we should merge #449 first and then double check what happens when rebasing, i.e. make sure nothing gets undone!